src/main/java/org/broadinstitute/barclay/help/BashTabCompletionDoclet.java

```diff
@@ -0,0 +1,356 @@
+package org.broadinstitute.barclay.help;
+
+import com.sun.javadoc.RootDoc;
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * For testing of help documentation generation.
+ */
+public class BashTabCompletionDoclet extends HelpDoclet {
+
+    // Barclay BashTabCompletionDoclet custom Command-line Arguments:
```

Comment:
It would also be good to document which, if any, are optional (currently I think its all of them, though I think the launch script name should be required).